requirements-dev.txt

```diff
@@ -98,8 +98,6 @@ molecule==3.5.2
     #   molecule-docker
 molecule-docker==1.1.0
     # via -r requirements-dev.in
-more-itertools==8.4.0
-    # via pytest
```

Comment:
This could be squashed into 14cdd11eb54db08bb9e4683adc1ba7f780091bfc, since it's `pytest==6.1.0` that removed the dependency on `more-itertools` (see https://github.com/pytest-dev/pytest/releases/tag/6.1.0). However I don't mind, so feel free to ignore.

Reply:
I'll leave it since we have a commit that's "re-compile requirements-dev.txt", and to me it's implied that this is re-compiling based on the changes that went before it on the branch.